practice-app/website/views.py

```diff
@@ -131,4 +133,23 @@ def view_event(event_id):
         weather_icon_url = f"http://openweathermap.org/img/wn/{icon}@2x.png"
         return render_template("event_page_by_id.html", user = current_user, event = event, weather_icon_url = weather_icon_url)
 
+      
+
+
+# Shows the discussion page for the event with id event_id. 
+# It also shows the description of the sport type of the event 
+# using an external API 
+@views.route('/events/<event_id>/discussions', methods=["GET"])
+@login_required
+def discussionPage(event_id):
+    
+    BASE = 'http://127.0.0.1:5000/'  #  should be changed
+    response = requests.get(BASE + '/api/v1.0/events/' + event_id + '/discussions')
+
+    if response.status_code == 201:
+        description = response.json()["description"]
+        text = response.json()["text"]
+        return render_template("discussionPage.html", event_id=event_id, definition = description, text = text.split('#')) 
+    else:
+        flash('Something went wrong', category='error')
```

Comment:
I think else should also return a render_template.

Reply:
In fact it should never go to else as long as the setting is ok, yet it can be useful for debugging.

Reply:
When the connection is lost for example what is shown?

Reply:
It's only for debugging. I don't think that there's a problem about this point.

Reply:
It seems the conversation is not progressing. Please explain your approach in advence. Also please answer the question.

Reply:
It has been changed as desired.